docs/proposals/YYYYMMDD-template.md

```diff
@@ -114,58 +80,65 @@ It should be possible to collect this information before implementation begins i
 A good summary is probably at least a paragraph in length.
 
 ## Motivation
-
 This section is for explicitly listing the motivation, goals and non-goals of this proposal.
-Describe why the change is important and the benefits to users.
-The motivation section can optionally provide links to [experience reports][] to demonstrate the interest in a proposal within the wider Kubernetes community.
 
-[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+- Describe why the change is important and the benefits to users.
+- The motivation section can optionally provide links to [experience reports](https://github.com/golang/go/wiki/ExperienceReports)
+to demonstrate the interest in a proposal within the wider Kubernetes community.
 
 ### Goals
 
-List the specific goals of the proposal.
-How will we know that this has succeeded?
+- List the specific goals of the proposal.
+- How will we know that this has succeeded?
 
-### Non-Goals
+### Non-Goals/Future Work
 
-What is out of scope for this proposal?
-Listing non-goals helps to focus discussion and make progress.
+- What is out of scope for this proposal?
+- Listing non-goals helps to focus discussion and make progress.
 
 ## Proposal
-
 This is where we get down to the nitty gritty of what the proposal actually is.
 
-### User Stories [optional]
+- What is the plan for implementing this feature?
+- What data model changes, additions or removals are required?
+- How does it impact user experience?
+- Provide a scenario, or example.
+- Use diagrams to communicate concepts, flows of execution, and states.
```

Comment:
This seems more in line with implementation details than the high level proposal description.

Reply:
Same as above